README.md

```diff
@@ -3,47 +3,47 @@
 Tools for searching issues on github. It can be filtered out only to return issues that suitable for you to contribute.
 
 ## TO DO
+
 See more at [Project#1](https://github.com/bekicot/gsoc-prep-tasks/projects/1)
+
 - Adding more link for GSOC Organizations. See [Add a easy selector for GSOC org](https://github.com/bekicot/gsoc-prep-tasks/issues/6).
 
 ## Prerequisites
 
 You will need the following things properly installed on your computer.
 
-* [Git](https://git-scm.com/)
-* [Node.js](https://nodejs.org/) (with NPM)
-* [Bower](https://bower.io/)
-* [Ember CLI](https://ember-cli.com/)
-* [PhantomJS](http://phantomjs.org/)
+- [Git](https://git-scm.com/)
```

Comment:
configure the bear to use `*` as the list item marker

Reply:
Which bear?

Reply:
Read the .coafile you created to find the answer to that question.

Reply:
this has not been done yet

Reply:
I was confused `*` with file glob. as you use `the bear` instead of `MarkdownBear`. Until realize it now.